examples/zlib_tokio_02_write.rs

```diff
@@ -0,0 +1,31 @@
+use async_compression::tokio_02::write::ZlibDecoder;
+use async_compression::tokio_02::write::ZlibEncoder;
+
+use std::io::Result;
+use tokio_02::io::AsyncWriteExt; // for `write_all` and `shutdown`
```

Comment:
```suggestion
use tokio_02::io::AsyncWriteExt as _; // for `write_all` and `shutdown`
```

Reply:
this will not work if I have 2 items like that

Reply:
You can have multiple things renamed to `_`.

Reply:
oh my bad it was an editor false negative that made me think I can't.
fixed it